sdk/template/template/rollup.base.config.js

```diff
@@ -52,13 +52,13 @@ export function nodeConfig(test = false) {
 export function browserConfig(test = false, production = false) {
   const baseConfig = {
     input: input,
-    external: ["ms-rest-js"],
+    external: ["@azure/ms-rest-js"],
```

Comment:
@ramya-rao-a, @bterlson: Do you know if this is correct?  I just did a blind update in the source code from "ms-rest-js" to "@azure/ms-rest-js".  FWIW, the build succeeded with or without this change in `rollup.base.config.js`.  However, the build failed without the changes in `index.ts`.

Reply:
Yes, that is correct.

This gets used only when we roll up for browser which we don't do yet. Thus build succeeded with/without the change.

The build would fail without the changes in index.ts file because without the change we would be referring to a package that doesnt exist, and rollup wouldnt like that